deeprvat/data/dense_gt.py

```diff
@@ -409,7 +409,18 @@ def transform_data(self):
                 self.phenotype_df[col] = rng.permutation(
                     self.phenotype_df[col].to_numpy()
                 )
-
+        if len(self.y_phenotypes) > 0:
+            unique_y_val = self.phenotype_df[self.y_phenotypes[0]].unique()
+            n_unique_y_val = np.count_nonzero(~np.isnan(unique_y_val))
+            logger.info(f"unique y values {unique_y_val}")
+            logger.info(n_unique_y_val)
+        else:
+            n_unique_y_val = 0
+        if n_unique_y_val == 2:
+            logger.info(
```

Comment:
Suggest to make this a warning, since a value from the config is being ignored

Reply:
Done